src/handlers/int-test-support/helpers/s3Helper.ts

```diff
@@ -132,6 +146,83 @@ const deleteS3Objects = async (
   );
 };
 
+/* Deletes s3 objects by prefixes in batches */
+export const batchDeleteS3ObjectsByPrefixesInBatchBasic = async (
```

Comment:
This name is far too long and unnecessarily complicated. No one cares if it uses batches as long as the objects are deleted in the quickest manner.

I propose `deleteS3ObjectsByPrefixes` or even just `deleteS3Objects`.